data.rs

```diff
@@ -52,6 +67,7 @@ impl PSP22Data {
         self.allowances.get((owner, spender)).unwrap_or_default()
     }
 
+    /// Transfer `value` tokens from `caller` to `to`.
```

Comment:
I'd be kind of tempted to rename `caller` to `from`. Caller is more meaningful, but

> from `from` to `to`

would be just the greatest line to have in the docs :smile: